src/Core/src/Core/IHybridWebView.cs

```diff
@@ -1,18 +1,31 @@
-﻿namespace Microsoft.Maui
+﻿using System.Threading.Tasks;
+
+namespace Microsoft.Maui
 {
 	public interface IHybridWebView : IView
 	{
+		/// <summary>
+		/// Specifies the file within the <see cref="HybridRoot"/> that should be served as the default file. The
+		/// default value is <c>index.html</c>.
+		/// </summary>
 		string? DefaultFile { get; }
 
 		/// <summary>
 		///  The path within the app's "Raw" asset resources that contain the web app's contents. For example, if the
 		///  files are located in <c>[ProjectFolder]/Resources/Raw/hybrid_root</c>, then set this property to "hybrid_root".
-		///  The default value is <c>HybridRoot</c>, which maps to <c>[ProjectFolder]/Resources/Raw/HybridRoot</c>.
+		///  The default value is <c>wwwroot</c>, which maps to <c>[ProjectFolder]/Resources/Raw/wwwroot</c>.
 		/// </summary>
 		string? HybridRoot { get; }
 
-		void RawMessageReceived(string rawMessage);
+		void MessageReceived(string rawMessage);
```

Comment:
Instead of having the processing done in Controls, it should be here with maybe an event args with the message type:

```cs
void MessageReceived(HybridWebViewMessageRecieved  message);
```

```cs
class HybridWebViewMessageRecieved {
}
```

```cs
class HybridWebViewRawMessageRecieved : HybridWebViewMessageRecieved {
}
```

```cs
class HybridWebViewInvokeMessageRecieved : HybridWebViewMessageRecieved {
}
```

This allows for easy extension and possibly better handler-specific implementations instead of parsing in XAML controls.